### PR TITLE
Adding labs and terminals to construction priority

### PR DESCRIPTION
**Pull request by Perrytheplatypus43**
This is adding labs and terminals in the construction priority

### CODE REVIEW

File: parameter.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ let mod = {
     NOTIFICATE_HOSTILES: true, // Log any hostiles - Ignores NOTIFICATE_INTRUDER and NOTIFICATE_INVADER
     COMBAT_CREEPS_RESPECT_RAMPARTS: false, // causes own creeps not to leave through ramparts
     COST_MATRIX_VALIDITY: 1000,
-    CONSTRUCTION_PRIORITY: [STRUCTURE_SPAWN,STRUCTURE_EXTENSION,STRUCTURE_LINK,STRUCTURE_TERMINAL,STRUCTURE_STORAGE,STRUCTURE_TOWER,STRUCTURE_ROAD,STRUCTURE_CONTAINER,STRUCTURE_EXTRACTOR,STRUCTURE_WALL,STRUCTURE_RAMPART],
+    CONSTRUCTION_PRIORITY: [STRUCTURE_SPAWN,STRUCTURE_EXTENSION,STRUCTURE_LINK,STRUCTURE_TERMINAL,STRUCTURE_STORAGE,STRUCTURE_TOWER,STRUCTURE_ROAD,STRUCTURE_CONTAINER,STRUCTURE_EXTRACTOR,STRUCTURE_WALL,STRUCTURE_RAMPART,STRUCTURE_TERMINAL,STRUCTURE_LAB],
```
Comment (chobobobo) [inline parameter.js]: Terminals are already part of the priority, but labs are a welcome addition, should they be the lowest priority?

Comment (Spedwards) [inline parameter.js]: They should come after the Extractors.

**Comment by Perrytheplatypus43** [comment]
I got rid of the terminal and put lads behind extractors.

**Comment by logxen** [comment]
While we're at it to powerSpawn, nuker, and observer belong on there somewher?

**Comment by Perrytheplatypus43** [comment]
@logxen i'll add them

**Comment by Spedwards** [comment]
Probably in the order of PowerSpawn, Nuker, Observer. I'd say after labs.

**Comment by Perrytheplatypus43** [comment]
I added PowerSpawn, Nuker, Observer before labs. Should i put them after labs?

**Comment by logxen** [comment]
actually should observer come before powerspawn? :o

**Comment by Spedwards** [comment]
It doesn't really matter. At the end of the day the player can viral the file and modify it as they like.

**Comment by Perrytheplatypus43** [comment]
Is this pull request ready to merge?

File: parameter.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ let mod = {
     NOTIFICATE_HOSTILES: true, // Log any hostiles - Ignores NOTIFICATE_INTRUDER and NOTIFICATE_INVADER
     COMBAT_CREEPS_RESPECT_RAMPARTS: false, // causes own creeps not to leave through ramparts
     COST_MATRIX_VALIDITY: 1000,
-    CONSTRUCTION_PRIORITY: [STRUCTURE_SPAWN,STRUCTURE_EXTENSION,STRUCTURE_LINK,STRUCTURE_TERMINAL,STRUCTURE_STORAGE,STRUCTURE_TOWER,STRUCTURE_ROAD,STRUCTURE_CONTAINER,STRUCTURE_EXTRACTOR,STRUCTURE_WALL,STRUCTURE_RAMPART],
+    CONSTRUCTION_PRIORITY: [STRUCTURE_SPAWN,STRUCTURE_EXTENSION,STRUCTURE_LINK,STRUCTURE_TERMINAL,STRUCTURE_STORAGE,STRUCTURE_TOWER,STRUCTURE_POWERSPAWN,STRUCTURE_NUKER,STRUCTURE_OBSERVER,STRUCTURE_ROAD,STRUCTURE_CONTAINER,STRUCTURE_EXTRACTOR,STRUCTURE_LAB,STRUCTURE_WALL,STRUCTURE_RAMPART],
```
Comment (karlthepagan) [inline parameter.js]: STRUCTURE_POWERSPAWN is not defined in prod

Comment (Perrytheplatypus43) [inline parameter.js]: should i get rid of STRUCTURE_POWERSPAWN

Comment (Spedwards) [inline parameter.js]: @Perrytheplatypus43 This was already sorted #933

Comment (Perrytheplatypus43) [inline parameter.js]: @Spedwards Ya I saw sorry

**Pull request merged**